tests/api/reports-import.php

```diff
@@ -320,7 +336,7 @@ public function test_import_status() {
 		);
 
 		$this->assertEquals( 200, $response->get_status() );
-		$this->assertEquals( $i, $report['orders'] );
+		$this->assertEquals( $i - 1, $report['orders'] );
```

Comment:
The `$i - 1` doesn't have any explanation for deducting the 1. This might be better with a comment or just `4` in the assertion.